infoschema/tables.go

```diff
@@ -1971,6 +1985,145 @@ func dataForTiDBClusterInfo(ctx sessionctx.Context) ([][]types.Datum, error) {
 	return rows, nil
 }
 
+func dataForClusterConfig(ctx sessionctx.Context) ([][]types.Datum, error) {
+	sql := "SELECT type, name, address, status_address FROM INFORMATION_SCHEMA.TIDB_CLUSTER_INFO ORDER BY type"
+	rows, _, err := ctx.(sqlexec.RestrictedSQLExecutor).ExecRestrictedSQL(sql)
```

Comment:
How about use `dataForTiDBClusterInfo` directly?

Reply:
The caller cannot assume the `dataForTiDBClusterInfo` results and prefer to use `ORDER BY` clause instead of sorting results manually.